### PR TITLE
feat(docs) steps on how to troubleshoot the slack command not working

### PR DESCRIPTION
**Pull request by jayacryl**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs/managed-datahub/slack/saas-slack-troubleshoot.md [hunk 1]
```diff
@@ -69,3 +69,24 @@ If you've subscribed to an entity, and then performed an action (i.e., raised an
 ### There is an issue with DataHub's systems
 If sending a test notification works, and you've verified that none of the above cases apply, then you should contact your DataHub Customer Success rep to help troubleshoot and resolve the issue. 
 
+## Slack bot issues
+This is to troubleshoot issues with your slack bot
+
+### Command failed with error "dispatch_failed"
+If you've installed the slack bot, but your commands are failing with an error 'dispatch_failed', you can try the following to correct it.
```
Comment (maggiehays) [inline docs/managed-datahub/slack/saas-slack-troubleshoot.md:76]: Nit: uppercase `Slack`

File: docs/managed-datahub/slack/saas-slack-troubleshoot.md [hunk 2]
```diff
@@ -69,3 +69,24 @@ If you've subscribed to an entity, and then performed an action (i.e., raised an
...
+1. Open your acryl instance with the following url: `<your-instance-base-url>/settings/integrations/slack?display_all_configs=true`.
```
Comment (maggiehays) [inline docs/managed-datahub/slack/saas-slack-troubleshoot.md:77]: Should this be DataHub Cloud instance instead?

File: docs/managed-datahub/slack/saas-slack-troubleshoot.md [hunk 3]
```diff
@@ -69,3 +69,24 @@ If you've subscribed to an entity, and then performed an action (i.e., raised an
...
+2. Switch to the **Bot Token** tab.
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/slack/verify_tokens_1.png"/>
+</p>
+3. The values will be `**`'d out, but if any of these fields are empty (ie. Signing Secret), then you'll want to fill them in.
```
Comment (maggiehays) [inline docs/managed-datahub/slack/saas-slack-troubleshoot.md:82]: You might need to put a linebreak after `</p>` & before #3 -- the rich diff isn't rendering the markdown/handling linebreaks accurately from here on out: 
CleanShot 2024-10-28 at 11 31 40@2x
CleanShot 2024-10-28 at 11 31 52@2x

**Comment by maggiehays** [comment]
@jayacryl left some comments, good to ship once those are addressed!

**Review by maggiehays — CHANGES_REQUESTED** [review]
left some comments, good to ship once those are addressed!

File: docs/managed-datahub/slack/saas-slack-troubleshoot.md (hunk 2 above)
Comment (jayacryl) [reply docs/managed-datahub/slack/saas-slack-troubleshoot.md:77]: haha good catch

File: docs/managed-datahub/slack/saas-slack-troubleshoot.md (hunk 3 above)
Comment (jayacryl) [reply docs/managed-datahub/slack/saas-slack-troubleshoot.md:82]: Ah weird. It looks okay when built though:
image
The linebreak adds extra space in the build

**Review by maggiehays — APPROVED** [review]
LGTM

**Pull request merged**